src/libraries/Common/src/Interop/Windows/Kernel32/Interop.LocalAlloc.cs

```diff
@@ -10,6 +10,7 @@ internal static partial class Kernel32
     {
         internal const uint LMEM_FIXED = 0x0000;
         internal const uint LMEM_MOVEABLE = 0x0002;
+        internal const uint LMEM_ZEROINIT = 0x0040;
 
         [DllImport(Libraries.Kernel32)]
```

Comment:
Should we just make these `GeneratedDllImport` too? Since we do the extern partial forwarder for blittable functions now.

Reply:
They are used in other projects so I didn't update them. I tried to only update P/Invokes that I knew worked everywhere. Some consumers don't have the needed references.